akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -166,7 +166,7 @@ private[remote] final class InboundActorRefCompression(
   originUid:      Long,
   inboundContext: InboundContext,
   heavyHitters:   TopHeavyHitters[ActorRef])
-  extends InboundCompression[ActorRef](log, settings, originUid, inboundContext, heavyHitters) {
+  extends InboundCompression[ActorRef](log, settings, originUid, inboundContext, heavyHitters) { // FIXME fix the logger
```

Comment:
was it fixed?

Reply:
Yeah, the logger is fine actually; was just blind when debugging when I added those prefixes